Singer.API/ClientApp/src/app/modules/admin/components/careusers/overview/overview.component.ts

```diff
@@ -68,10 +68,22 @@ export class OverviewComponent implements OnInit, AfterViewInit {
 
       dialogRef.componentInstance.submitEvent.subscribe((result: CareUser) => {
          // Update the Careuser
-         this.careUserService.updateUser(result).subscribe(res => {
```

Comment:
Remember our discussion regarding javascript and by reference passing? It seems that javascript does this for objects and arrays, and only (obviously) for primitive types by value. So yes, updating the row values (even in a popup), means actually updating the row.

Reply:
It has been fixed